topology/Texas Advanced Computing Center/TACC - Frontera/TACC-Frontera.yaml

```
@@ -0,0 +1,118 @@
+# Production is true if the resource is for production and not testing use
+Production: true
+# SupportCenter is one of the support centers in topology/support-centers.yaml
+SupportCenter: Self Supported
+
+# GroupDescription is a long description of the resource group; may be multiple lines.
+GroupDescription: TACC-Frontera cluster
+
+# If you have an up-to-date local git clone, fill GroupID with the output from `bin/next_resource_group_id`
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+GroupID: 1099
+
+# Resources contains one or more resources in this
+# ResourceGroup. A resource provides one or more services
+Resources:
+  # Resource Name should be a short descriptor of the resource.
+  # e.g. the Center for High Throughput Computing's GlideinWMS Frontend is "CHTC-glidein2"
+  # Resource Names need to be unique across all resources in the OSG.
+  TACC-Frontera:
+    # Active is true if the resource is accepting requests, and false otherwise.
+    # When first registering a resource, set this to false. Set it to true when it's ready for production.
+    Active: true
+    # Description is a long description of the resource; may be multiple lines
+    Description: Hosted CE serving TACC-Frontera:
```

```suggestion
    Description: Hosted CE serving TACC-Frontera
```
a string with a comma needs to be quoted (but in this case I don't think you meant to put a comma there)